randovania/games/dread/generator/bootstrap.py

```diff
@@ -40,8 +40,7 @@ def create_damage_state(self, game: GameDescription, configuration: DreadConfigu
         return EnergyTankDamageState(
             configuration.energy_per_tank - 1,
             configuration.energy_per_tank,
-            game.resource_database,
-            game.region_list,
+            game.resource_database.energy_tank,
         )
```

Comment:
Doesn't this need to be `DreadDamageState` now? To me it looks like `DreadDamageState` is complety unused, so it is actually missing the key part of this PR?

Reply:
Somehow forgot to add that back in after managing to commit after the precommit hook conflict fiasco 😅